src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -494,8 +494,7 @@ private async Task<bool> ShouldExitAfterDeterminingHostRuntime(bool isVerbose)
 
                 if (!string.Equals(HostRuntime, "default", StringComparison.OrdinalIgnoreCase))
                 {
-                    var isNet8InProcSpecified = string.Equals(HostRuntime, DotnetConstants.InProc8HostRuntime, StringComparison.OrdinalIgnoreCase);
-                    StartHostAsChildProcess(isNet8InProcSpecified);
+                    PrintMessageForInProcSdk();
```

Comment:
do we always want this message printed out? Or only if we see it's not the lastest version?

Reply:
If they are using the latest version, the payload would not reach here.

Reply:
Can you help me understand that? It seems like this is a code path that comes in naturally by simply running `func start --runtime inproc8`.

Maybe I'm tracing through it wrong, but it seems like that would be `RunAsync()` -> `ShouldExitAfterDeterminingHostRuntime()`. The `ValidateHostRuntime` method would return without throwing if I was indeed in the context of a .NET 8 in-process project. And then I would fail the string equality check against "default" and end up here.

That means this would break standard command line usage. I can't find the preconditions which assert this to be the VS variant or determine if the payload is missing the required SDK version.

Reply:
Yeah so based on my discussion with Fabio, there does not seem to be any branching in code that one path will for VS scenario and other part will go to command line. Its the same code and during release, we use some configuration to prepare coretools for VS scenario.

Reply:
I was able to confirm in our runtime sync. It is during the release we make the distinction.

Reply:
Will add more changes here which checks for VS variant